src/Take.Blip.Client/Extensions/ContactsJourney/ContactsJourneyExtension.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Takenet.Iris.Messaging.Resources.Analytics;
+
+namespace Take.Blip.Client.Extensions.ContactsJourney
+{
+    public class ContactsJourneyExtension : ExtensionBase, IContactsJourneyExtension
+    {
+        private const string CONTACTS_JOURNEY_URI = "/contacts-journey";
+
+        private readonly Node AnalyticsAddress = Node.Parse("postmaster@analytics.msging.net");
```

Comment:
We have a constants file with the Domain  (  see HelpDeskExtension for an example )

Reply:
Done